source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -226,19 +226,28 @@
                     name: groupNames[key],
                     items: _.map(items, function (item) {
                         itemNumber++;
+                        var rowSelected = ko.observable(false);
                         var valuesByItem = {
                             key: item.Key,
                             text: itemNumber + " - " + item.Text,
                             description: item.Description,
+                            isRowSelected: rowSelected,
                             values: _.map(valuesByKeyCollection, function (valuesByKey) {
                                 var valueItem = {
                                     calificationId: valuesByKey.calificationColumn.calificationId,
                                     value: ko.observable(valuesByKey[item.Key] || ""),
                                     editable: self.isValueEditable(valuesByKey),
+                                    isSelected: rowSelected,
                                     owner: valuesByKey.calificationColumn.owner,
                                     showValue: _.find(self.califications, function (calification) {
                                         return calification.id == valuesByKey.calificationColumn.calificationId;
                                     }).show,
+                                    onBlur: function (data, event) {
+                                        data.isSelected(false);
+                                    },
+                                    onFocus: function (data, event) {
+                                        data.isSelected(true);
+                                    }
```

Comment:
@AlphaGit  JD, talking with Jose about improve the code, we decided to do this improve but in the future, because I need some time to do this without breaking something and there are lot of things with more priority. Sound good to you JD??